core/arch/arm/kernel/generic_boot.c

```diff
@@ -75,6 +75,7 @@
 
 static uint8_t pseudo_device_id[160] __early_bss __attribute__ ((aligned (8)));
 static size_t pseudo_device_id_len __early_bss;
+static const uint8_t default_pattern[4] = { 'B', 'E', 'E', 'F' };
```

Comment:
Why move this out of `tee_otp_get_die_id()`? It's not referenced from anywhere else.